src/models/position.js

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
Do we need both of these classes? Can we consolidate them into a Vector or something?

Reply:
Also please fix the indent levels